### ISSUE TITLE
Support an option for the trigger characters?

### ISSUE THREAD

**Issue by stevenproctor** [hrsh7th/cmp-emoji#6]
I enjoy the plugin, but I also work in a few languages that make frequent use of colons as starting characters for keywords/symbols.  What are your thoughts on making an option for the trigger characters?

**Comment by abhaysp95** [comment]
Yeah, it becomes hell to work with languages like python. The help page doesn't tell how to setup the trigger character
